index.md

```diff
@@ -134,7 +134,7 @@ There are a few libraries and demos available to make connecting to our API easi
 
 ## Handy tools
 
-*   [Hookbin](https://hookbin.com) for testing the postback.
+*   [Webhook Tester](https://webhook.site) / [Hookbin](https://hookbin.com) for testing the postback.
```

Comment:
Just remove hookbin.com altogether. perhaps add http://webhookinbox.com/ as an extra option (although it doesn't support https)